src/main/java/com/dev/theater/service/impl/ShoppingCartServiceImpl.java

```diff
@@ -0,0 +1,46 @@
+package com.dev.theater.service.impl;
+
+import com.dev.theater.dao.ShoppingCartDao;
+import com.dev.theater.dao.TicketDao;
+import com.dev.theater.library.Inject;
+import com.dev.theater.library.Service;
+import com.dev.theater.model.MovieSession;
+import com.dev.theater.model.ShoppingCart;
+import com.dev.theater.model.Ticket;
+import com.dev.theater.model.User;
+import com.dev.theater.service.ShoppingCartService;
+
+@Service
+public class ShoppingCartServiceImpl implements ShoppingCartService {
+    @Inject
+    private ShoppingCartDao shoppingCartDao;
+    @Inject
+    private TicketDao ticketDao;
+
+    @Override
+    public void addSession(MovieSession movieSession, User user) {
+        Ticket ticket = new Ticket();
+        ticket.setMovieSession(movieSession);
+        ticket.setUser(user);
+        ticketDao.add(ticket);
+        shoppingCartDao.update(shoppingCartDao.getByUser(user));
```

Comment:
what are you doing exactly in this method?

Reply:
Idk, changed